src/server/system_services/schemas/bucket_schema.js

```diff
@@ -319,6 +319,9 @@ module.exports = {
                     object_suffix: {
                         type: 'string'
                     },
+                    attempts: {
```

Comment:
@liranmauda i think the field should be called `retries` in the API too

Reply:
The API is with Attempts and not retries as it reflects the behavior better.
We are using `P.retry` and it uses attempts. 
Providing 1 will run it once without retries, providing more will do retries.
https://github.com/noobaa/noobaa-core/blob/de261f32fb59a973fb244e84512f572be0a892f4/src/util/promise.js#L191-L195
I have changed the UI to be more straightforward, The BE left with attempts.

Reply:
ok